build_helper.sh

```diff
@@ -58,9 +58,6 @@ function prepare_upload_data {
   rm -rf ${LOCAL_PATH}
   mkdir -p ${LOCAL_PATH}
   mv handler.zip ${LOCAL_PATH}/lambda.zip
-}
-
-function prepare_cloudformation_code {
```

Comment:
The function's code is needed for the versioning, but it was just never called due to a human error, so I decided to merge its code into the function above it that is getting called. Deleting those three lines simply merges the code to the function above it.

This stuff used to be in the Makefile before, but I thought it would be cleaner to have it in a dedicated shell script.

This is how the Makefile used to look before I changed it a few weeks back: https://github.com/cristim/autospotting/blob/d51b65b01f71f152981388a44e7cab33c742d0a5/Makefile